src/coreComponents/integrationTests/xmlTests/testXMLFile.cpp

```diff
@@ -31,9 +31,13 @@ using namespace geos;
 using namespace geos::dataRepository;
 using namespace xmlWrapper;
 
+std::unique_ptr< CommandLineOptions > globalCommandLine = nullptr;
+
 // Tests if the xml file parsing works with one file, one file with nested includes, and multiple files
 TEST( testXML, testXMLFile )
 {
+  GeosxState state( std::make_unique< CommandLineOptions >( *globalCommandLine ) );
```

Comment:
Having a `GeosxState` for each tests prevents duplicate registration from repeatedly reading in XML files. This is probably what we want to test anyways.